### PR TITLE
Use JDK 11 when running nightly

### PR DESCRIPTION
**Pull request by 2m**
## Purpose

Switches to JDK 11 when running nightly.

### CODE REVIEW

**Comment by ennru** [comment]
Fixes #1697

**Comment by 2m** [comment]
Unrelated failure were #1649

File: .travis.yml [hunk 1]
```diff
@@ -10,7 +10,9 @@ before_install:
 
 # default script for jobs, that do not have any specified
 script:
-  - ./scripts/buildChanged.sh ${DIR:=.} "${JDK:=adopt@~1.8.202-08}" "${PRE_CMD:=echo NOOP}" "${CMD:=+$DIR/testChanged}"
+  - |
+    [ "$TRAVIS_EVENT_TYPE" == "cron" ] && JDK="adopt@~1.11.0-2" || JDK="adopt@~1.8.202-08"
```
Comment (ennru) [inline .travis.yml]: What does the `|` do?

Comment (2m) [reply .travis.yml]: It switches to YAML template mode. Otherwise yaml parser is not happy about `[` and `]` characters.

**Pull request merged**